app/api.py

```diff
@@ -45,34 +44,27 @@ def create_warrior():
     if "name" not in data or "dob" not in data or "fight_skills" not in data:
         return jsonify({"message": "Bad Request - Missing required fields"}), 400
 
+    # Validate fight_skills
+    validation_error = validate_fight_skills(data['fight_skills'])
+    if validation_error:
+        return jsonify({"message": validation_error}), 400 
+
+    # Check the name is not too long
+    if len(data['name']) > 100:
+        return jsonify({"message": "Bad Request - name is too long"}), 400
+    
     id = str(uuid.uuid4())
     name = data.get("name")
     dob = data.get("dob")
     fight_skills = data.get("fight_skills")
 
-    # Check the name is more than 100 characters
-    if len(name) > 100:
-        return jsonify({"message": "Bad Request - name is too long"}), 400
     # Check for empty skills
-    if fight_skills == None:
+    if fight_skills is None:
         return jsonify({"message": "Bad Request - fight_skills cannot be empty"}), 400
+    
     # Check for more than 20 fight skills
     if len(fight_skills) > 20:
-        return (
-            jsonify(
-                {
-                    "message": "Bad Request - fight_skills cannot cannot have more than 20 skills"
-                }
-            ),
-            400,
-        )
-    # Check for skills that are more than 250 characters
-    if any(len(fight_skill) > 250 for fight_skill in fight_skills):
-        return jsonify({"message": "Bad Request - a fight skill name is too long"}), 400
-    # Check for total sum of the fight skill character lengths doesn't exceed max length
-    max_skills_length = 5019
-    if sum([len(fight_skill) for fight_skill in fight_skills]) > max_skills_length:
-        return jsonify({"message": "Bad Request - fight skills length exceeded"}), 400
+        return jsonify({"message": "Bad Request - fight_skills cannot have more than 20 skills"}), 400
```

Comment:
These lines should be removed if you're putting in your own fight_skills validator function.

Reply:
removed the unnecessary lines